docs/reference/using-logstash-with-elastic-integrations.md

```diff
@@ -74,6 +74,7 @@ output { <3>
 1. Use `filter-elastic_integration` as the first filter in your pipeline
 2. You can use additional filters as long as they follow `filter-elastic_integration`
 3. Sample config to output data to multiple destinations
+4. The format of the value is `id:api_key`, where `id` and `api_key` are the values returned by the [Create API key API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-security-create-api-key) or the [Get API key information API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-security-get-api-key)
```

Comment:
Let's remove the GET API key information API mention. The GET API will not return the actual api_key (for security reasons). Only the Create API key will return it at creation time.